keyboards/exclusive/e6v2/le_bmc/rules.mk

```diff
@@ -0,0 +1,90 @@
+# MCU name
+#MCU = at90usb1286
```

Comment:
```suggestion
```

Reply:
We need to remove this from the templates STAT!

Reply:
#5884 😉